client/my-sites/preview/controller.js

```diff
@@ -12,6 +12,6 @@ import React from 'react';
 import PreviewMain from './main';
 
 export function preview( context, next ) {
-	context.primary = <PreviewMain site={ context.params.site } />;
+	context.primary = <PreviewMain site={ context.params.site } open={ context.query.open } />;
```

Comment:
Thanks for the explanation!

> There are no plans to have another value for help so we could do as you suggest, if you think that makes more sense.

Not a huge deal, I just thought it would reduce confusion as to the parameter's purpose to a newcomer.